packages/core/tsconfig-demo.json

```diff
@@ -22,7 +22,8 @@
 		"typeRoots": [
 			"./node_modules/@types",
 			"./custom.d.ts"
-		]
+		],
+		"allowSyntheticDefaultImports": true
```

Comment:
We shouldn't allow this, since it causes entire bundles to be dragged into our codebase